core/errors.rs

```diff
@@ -56,13 +56,13 @@ impl Deref for ErrBox {
 
 impl<T: AnyError> From<T> for ErrBox {
   fn from(error: T) -> Self {
-    Self(Box::new(error))
+    Self(Box::new(error), "Other")
   }
 }
```

Comment:
@ry our yesterday's approach is feasible when using `specialization` - unfortunately it's only available in nightly Rust